en/docs/concepts/provisioning-framework.md

```diff
@@ -11,17 +11,17 @@ The identity provisioning framework is the main component that handles user / ro
 
 ---
 
-## Inbound provisioning component
+## Inbound provisioning
 
-The inbound provisioning component focuses on how to provision users to WSO2 Identity Server. By default, WSO2 IS supports inbound provisioning via a Simple Object Access Protocol (SOAP) based API as well as the System for Cross-domain Identity Management (SCIM) API therefore, provisioning requests can come in the form of SCIM or SOAP. Both APIs support HTTP Basic Authentication. 
+Inbound provisioning focuses on how to provision users to WSO2 Identity Server. By default, WSO2 IS supports inbound provisioning via a Simple Object Access Protocol (SOAP) based API as well as the System for Cross-domain Identity Management (SCIM) API therefore, provisioning requests can come in the form of SCIM or SOAP. Both APIs support HTTP Basic Authentication. 
```

Comment:
users to WSO2 IS.

Reply:
what's wrong with calling it WSO2 Identity Server?

Reply:
I thought once we introduce the reader to the abbreviation in the beginning of a doc, we stick to the abbreviated version. Otherwise don't you think juggling between WSO2 IS and WSO2 Identity Server is inconsistent?